app/views/finders/show.html.erb

```diff
@@ -56,7 +72,9 @@
 </header>
 
 <div class="grid-row">
-  <form method="get" action="<%= finder.slug %>" class="js-live-search-form">
+  <% if finder.name != 'All content' %>
```

Comment:
`if finder.name != 'All content'` appears in quite a few places. Can we define it as a function in one place, e.g: the finder presenter?

Reply:
Good idea, I'll see what I can do